com.chartboost.mediation.demo/Assets/Demo/Demo.cs

```diff
@@ -3,10 +3,10 @@
 using System.IO;
 using System.Diagnostics.CodeAnalysis;
 using System.Text;
-using Helium;
-using Helium.Banner;
-using Helium.FullScreen.Interstitial;
-using Helium.FullScreen.Rewarded;
+using Chartboost;
```

Comment:
```suggestion
using ChartboostMediation;
```

Shouldn't this be ChartboostMediation??